client/src/components/Intro/index.tsx

```diff
@@ -70,24 +67,6 @@ const Intro = ({
         ) : (
           ''
         )}
-        <Alert bsStyle='info' className='annual-donation-alert'>
-          <p>
-            <b>{t('learn.season-greetings-fcc')}</b>
-          </p>
-          <p>{t('learn.if-getting-value')}</p>
```

Comment:
A quick note that we should probably remove these keys from the English JSON too